.github/workflows/multiflow.yml

```
@@ -2,61 +1,86 @@
+    - name: 🧱 Setup PHP
       uses: shivammathur/setup-php@v2
       with:
         php-version: '8.3.9'
-    - name: Cache Composer packages
+        extensions: mbstring, intl, zip, pdo_mysql
+        coverage: xdebug
+        tools: composer:v2, phpunit, phpcs, phpstan
+    
+    - name: 🎣 Récupération répertoire de cache composer
+      id: composer-cache
+      run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:50: Double quote to prevent globbing and word splitting

(shellcheck)